webpack.js

```diff
@@ -1,19 +1,11 @@
-const { merge } = require('webpack-merge')
 const path = require('path')
 const webpackConfig = require('@nextcloud/webpack-vue-config')
 
-const config = {
-	entry: {
-		admin: path.join(__dirname, 'src', 'admin.js'),
-		user: path.join(__dirname, 'src', 'user.js'),
-		public: path.join(__dirname, 'src', 'public.js'),
-		registration: path.join(__dirname, 'src', 'registration.js'),
-	},
-	output: {
-		filename: 'terms_of_service_[name].js',
-	},
-}
+delete webpackConfig.entry.main
+webpackConfig.entry.admin = path.join(__dirname, 'src', 'admin.js')
+webpackConfig.entry.user = path.join(__dirname, 'src', 'user.js')
+webpackConfig.entry.public = path.join(__dirname, 'src', 'public.js')
+webpackConfig.entry.registration = path.join(__dirname, 'src', 'registration.js')
+webpackConfig.output.filename = 'terms_of_service_[name].js'
```

Comment:
I would have prefered not, but at least you're using the appname as prefix :shrug:

Reply:
well minimal impact PRs stacking up. Can clear it at some point when all the places are adjusted